tests/containers/rootless_podman.pm

```diff
@@ -170,13 +170,16 @@ sub check_bsc1200623() {
     # cid = "
     # 2022-07-05T08:48:56.151176-04:00 susetest systemd[3438]: Failed to start podman-6734.scope.
     # 5b08b0dc136dd32bb30e69e4deb5df511dea0602d6b0c8d3623120370184506a"
-    # So, we need to remove the first one.
-    if ($cid =~ /Failed to start podman/) {
-        record_soft_failure('bsc#1200623 - systemd[3557]: Failed to start podman-3627.scope') unless ($bsc1200623);
-        $bsc1200623 = 1;    # to prevent printing the soft-failure more than once
-        ($cid) =~ s/.*\n//;
+    # So, we need to remove everything except the container ID.
+    if ($cid =~ /^([0-9a-f]{64})\s*$/) {
+        return $1;
```

Comment:
Shouldn't here be purged the string part other than cid, adding:
```suggestion
      ($cid) =~ s/.*\n//;
```
?

Reply:
I just reworked the whole logic. Please review again.